gameCode.cpp

```diff
@@ -0,0 +1,155 @@
+#include <iostream>
```

Comment:
Please try to create classes for Snake and Ladder as well at least

Reply:
Done